lib/src/boilerplate_suggestors/boilerplate_utilities.dart

```diff
@@ -406,8 +466,9 @@ class ClassToMixinConverter {
     if (node.extendsClause?.extendsKeyword != null) {
       // --- Convert concrete props/state class to a mixin --- //
 
-      yieldPatch(node.name.token.offset,
-          node.name.token.offset + privateGeneratedPrefix.length, '');
+      if (node.name.name != newMixinName) {
+        yieldPatch(node.name.token.offset, node.name.token.end, newMixinName);
+      }
```

Comment:
This doesn't seem related to the semver stuff... is this intentional, or a bad merge possibly?